apps/publish/content/common.py

```diff
@@ -895,15 +895,32 @@ def _publish_associated_items(self, original, updates=None):
                         sync_associated_item_changes(associated_item, associated_item_updates)
                         continue
 
-                    if association_updates.get("state") not in PUBLISH_STATES:
-                        # There's an update to the published associated item
+                    orig_associated_item = archive_service.find_one(req=None, _id=associated_item[config.ID_FIELD])
+                    if self.is_changed(orig_associated_item, associated_item):
```

Comment:
I wonder if it would publish media item in progress if there were no changes

Reply:
should we explicitly skip calling `publish_service.patch(...`) for associated items that are still in `in_progress` (or other non-published) states?

Reply:
I think the issue might be rather the other way around, that it wouldn't get called when it should be.
maybe we should have something like this:
```py
 if association_updates.get("state") not in PUBLISH_STATES or self._is_changed(...):
 ```
but I wonder if it actually gets there if no tests were failing

Reply:
makes sense. do you have a specific test scenario in mind that we might be missing here? I can add that case to the tests and we’ll see if the current logic gets triggered and behaves as expected

Reply:
just publishing an item with image in progress without any changes made during publishing

Reply:
added a new test case